docs/UserGuide.md

```diff
@@ -97,7 +97,15 @@ Examples:
 
 Adds an internship role to a company that already exists in the list.
 
-Format: `addRole COMPANY_INDEX n/ROLE s/STATUS b/DEADLINE d/DESCRIPTION $/STIPEND`
+Format: `addRole COMPANY_INDEX n/ROLE_NAME s/STATUS b/DEADLINE [d/DESCRIPTION] [$/STIPEND]`
+* Add internship role at the specified `COMPANY_INDEX`.
+* The `COMPANY_INDEX` must be a positive integer like 1, 2, 3, ...
+* The `ROLE_NAME` should only contain alphanumeric characters and spaces.
+* The `STATUS` accepted are as follows: applying, pending, offered, rejected, complete.
+* The `DEADLINE` should be in format dd-MM-yyyy HH:mm
+* The `DESCRIPTION` and `STIPEND` fields are optional during the initial role creation
+* The `DESCRIPTION` can contain alphanumeric characters, spaces and special characters.
+* The `STIPEND` must be a positive integer going up to 10 digits long.
```

Comment:
Looks great overall 👍

Trivial but I think the last two bullet points (107-108) on the `DESCRIPTION` and `STIPEND` can be indented to illustrate that they are a part of the third last bullet point (106).

Reply:
Noted, updated in new commit